.github/workflows/vulture.yml

```diff
@@ -19,5 +19,5 @@ jobs:
         uses: anaynayak/python-vulture-action@v1.0
         id: vulture
         with:
-          vulture-args: --min-confidence 80 --ignore-names cls,args,kwargs ${{steps.files.outputs.all}}
+          vulture-args: --min-confidence 80 --ignore-names cls,args,kwargs,real_variant_database ${{steps.files.outputs.all}}
```

Comment:
One could just save it to something local as well, or disable quality checking for those lines, but sure, why not!

Reply:
This looked easier 😆